src/views/virtualmachines/details/tabs/configuration/scripts/components/DynamicSSHKeyInjectionDescription.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { DYNAMIC_SSH_KEY_INJECTION_LINK } from '@kubevirt-utils/components/DynamicSSHKeyInjection/constants/constants';
+import { t } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Button, StackItem } from '@patternfly/react-core';
+import { ExternalLinkSquareAltIcon } from '@patternfly/react-icons';
+
+const DynamicSSHKeyInjectionDescription = () => {
+  return (
+    <div>
+      <StackItem>
+        <div className="text-muted">
+          {t('Dynamic SSH key injection is not enabled in this virtual machine.')}{' '}
+          <Button
+            icon={<ExternalLinkSquareAltIcon />}
+            iconPosition="right"
+            isInline
+            isSmall
+            variant="link"
+          >
+            <a href={DYNAMIC_SSH_KEY_INJECTION_LINK} rel="noopener noreferrer" target="_blank">
```

Comment:
instead of rendering this explicitly, you can add `component="a"` prop to the button along with `href, rel and target` props